.github/workflows/R-CMD-check.yaml

```diff
@@ -5,10 +5,12 @@ on:
     branches:
       - main
```

Comment:
`main` not needed

Reply:
Agree. `main` was just there in the template, since it's an alternative name to `master`, so best to remove it.